vsutil/__init__.py

```diff
@@ -2,7 +2,8 @@
 VSUtil. A collection of general-purpose VapourSynth functions to be reused in modules and scripts.
 """
 __all__ = ['Dither', 'Range', 'depth', 'fallback', 'frame2clip', 'get_depth', 'get_plane_size',
-           'get_subsampling', 'get_w', 'get_y', 'insert_clip', 'is_image', 'iterate', 'join', 'plane', 'split']
+           'get_subsampling', 'get_w', 'get_y', 'insert_clip', 'is_image', 'is_variable', 'iterate', 'join', 'plane',
+           'split']
```

Comment:
Off-Topic, but:

This is ruby-code-style and this is horrible. Python demands:

```py
my_list = [
    "a",
    "b",
    ...
]

Reply:
I just auto-formatted the list with PyCharm, we can re-format this in another PR.
```py
__all__ = [
    'Dither', 'Range',

    'depth',
    'fallback',
    'frame2clip',
    'insert_clip',
    'iterate',
    'join',
    'plane',
    'split',

    'get_depth', 'get_plane_size', 'get_subsampling', 'get_w', 'get_y',

    'is_image', 'is_variable'
]
```
Would this be acceptable? Tt groups a few of them logically and looks better, if not we can one-per-line the whole list.

Reply:
I'm good with grouping personally, but the larger section in the middle could also be grouped in some ways.
Just remember to add a trailing comma.

Reply:
I'm fine with grouping too.

But formatting lists like this:
```py
my_list = ["123", "456", ...,
                "abc", "def"]
```

Is in my experience far too unstable, that's why I hate that syntax.